Update tool MWA to 0.0.1+galaxy0

dsavchenko opened the pull request.

Commit:
update tool MWA

Commit:
automatic update

following https://gitlab.renkulab.io/astronomy/mmoda/mwa/-/commit/566d0ae12a9d32a7a07184283bd62d3042f1d8f4

volodymyrss commented:
@dsavchenko , do you know why this one is not updating and also failing like that:

![image](https://github.com/esg-epfl-apc/tools-astro/assets/3909535/3960f619-82a1-4394-8e07-c092a4b441b9)

dsavchenko commented:
It's updated. But what's going on with matplotlib dependency, that I don't understand for a moment

dsavchenko committed:
Merge branch 'main' into auto-update-galaxy-tool-mwa-v0.0.1-galaxy0

dsavchenko commented:
Other weirdness is why karabo script is in the PR...

Commit:
automatic update

following https://gitlab.renkulab.io/astronomy/mmoda/mwa/-/commit/79aa0c527c281a49b5594355af05244412799768

dsavchenko committed:
manually delete unrelated karabo folder

Commit:
automatic update

following https://gitlab.renkulab.io/astronomy/mmoda/mwa/-/commit/f57c65247fb032896166ef24ecb58448a5610ab5

dsavchenko committed:
manually delete unrelated karabo folder

dsavchenko commented:
> Other weirdness is why karabo script is in the PR...

Here is a fix https://github.com/oda-hub/oda-bot/pull/68

It happened because karabo uses non-standard conda channels, so that converter can't convert it and produces exception. But a script is already generated at that point. In git it's untracked, so stays there while changing branch

dsavchenko commented:
> @dsavchenko , do you know why this one is not updating and also failing like that:

The problem was with instance config, namely having both `conda-forge,defaults` in channels leaded to libmamba exception. And galaxy silently gives up installing requirement in this case, so the environment didn't contain needed packages. 

It works now

Commit:
automatic update

following https://gitlab.renkulab.io/astronomy/mmoda/mwa/-/commit/e17eaa635b55e80e7fa51acc9a90112bb1926f83

Commit:
automatic update

following https://gitlab.renkulab.io/astronomy/mmoda/mwa/-/commit/e17eaa635b55e80e7fa51acc9a90112bb1926f83

Commit:
automatic update

following https://gitlab.renkulab.io/astronomy/mmoda/mwa/-/commit/41c2685a9d059d6a04503c2df4d3d1144fca0eb4

Commit:
automatic update

following https://gitlab.renkulab.io/astronomy/mmoda/mwa/-/commit/2237020e0b315562a941f5cbeeb57a550ea9f2c7

Commit:
automatic update

following https://gitlab.renkulab.io/astronomy/mmoda/mwa/-/commit/8227629d0244793c9a1d8072c36f5e7149975987

Commit:
automatic update

following https://gitlab.renkulab.io/astronomy/mmoda/mwa/-/commit/ad71bd9af2f40e9deb5046c10d571821ffdbd8a6

Commit:
automatic update

following https://gitlab.renkulab.io/astronomy/mmoda/mwa/-/commit/3f4b0d4906801c4e233f743cff9701299c1d2f82

Commit:
automatic update

following https://gitlab.renkulab.io/astronomy/mmoda/mwa/-/commit/5e365aa8fb0dae4a4b1043cb7c92bf89931a6c14

Commit:
automatic update

following https://gitlab.renkulab.io/astronomy/mmoda/mwa/-/commit/2c5db6400a25cc74067dc25f6ff483b9880b4afa

Commit:
automatic update

following https://gitlab.renkulab.io/astronomy/mmoda/mwa/-/commit/2759718e556079a9cf0490f33163f19d26fead47

Commit:
automatic update

following https://gitlab.renkulab.io/astronomy/mmoda/mwa/-/commit/fae2c134d611080d555896ee51d62ab1ecf12ce3

Commit:
automatic update

following https://gitlab.renkulab.io/astronomy/mmoda/mwa/-/commit/fae2c134d611080d555896ee51d62ab1ecf12ce3
